frictionless_ckan_mapper/frictionless_to_ckan.py

```diff
@@ -88,6 +89,7 @@ def package(fddict):
     if 'licenses' in outdict and outdict['licenses']:
         outdict['license_id'] = outdict['licenses'][0].get('name')
         outdict['license_title'] = outdict['licenses'][0].get('title')
+        outdict['license_url'] = outdict['licenses'][0].get('path')
```

Comment:
Frictionless to CKAN was pretty much working as expected but missing one possible key, `license_url`.

Reply:
You don't usually need to annotate your own commits - this stuff should be obvious from commit message (and PR summary).